core/src/main/java/jenkins/security/LastGrantedAuthoritiesProperty.java

```diff
@@ -148,10 +148,6 @@ protected void loggedOut(@Nonnull String username) {
 
     @Extension
     public static final class DescriptorImpl extends UserPropertyDescriptor {
-        public String getDisplayName() {
-            return null;    // not visible
```

Comment:
Though in this case it seems like the code may be relying on `f:section` suppressing the block. I will check it.